packages/bridge-ui-v2/src/components/Bridge/Recipient.svelte

```diff
@@ -32,12 +32,13 @@
   function openModal() {
     modalOpen = true;
     addressInput.focus();
+    addEscKeyListener();
   }
 
   function cancelModal() {
     // Revert change of recipient address
     $recipientAddress = prevRecipientAddress;
-
+    removeEscKeyListener();
```

Comment:
Same here, we could also add this onDestroy, just to make sure that when we are unmounting the whole Bridge component we are also destroying this listener

Reply:
I will add the call to onDestroy